src/libs/ajax.js

```diff
@@ -883,17 +883,17 @@ const Buckets = signal => ({
     return res.json()
   },
 
-  listAll: async (namespace, bucket, pageToken = null) => {
+  listAll: async (namespace, bucket, prefix = null, pageToken = null) => {
```

Comment:
Since I introduced this method, the uploader should be the only code using it so this is safe.